libs/components/src/lib/signup-form/signup-form.tsx

```diff
@@ -0,0 +1,166 @@
+import { useEffect } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Col, Form, Row, Space, Typography } from 'antd';
+import { useRegisterUserMutation } from 'src/redux/auth/auth-api';
+import { setUser } from 'src/redux/auth/auth-slice';
+import * as yup from 'yup';
+
+import { StyledInput, StylesButton } from '../login-form/styles';
+
+import 'antd/dist/antd.css';
```

Comment:
it should be one import for full project

Reply:
done

Reply:
what done, why I still see it here

Reply:
@lyudad I removed StyledInput and StyledButton from both files, didn't understand that you meant antd.css